src/staging/src/pages/active-advisories.js

```diff
@@ -338,7 +338,7 @@ const PublicAdvisoryPage = ({ data }) => {
   return (
     <>
       <Helmet>
-        <title>BC Parks | Public Advisories</title>
+        <title> Active advisories | BC Parks </title>
```

Comment:
Sorry for nitpicking, I'm just wondering if you could remove spaces between title tags. It wouldn't cause any issues but would look nicer IMHO.

Reply:
Actually, it's a good point. It might affect the page title in browser search results. Good catch!